src/plugins/intel_cpu/src/config.cpp

```diff
@@ -273,7 +273,9 @@ void Config::readProperties(const std::map<std::string, std::string> &prop) {
 
 #if defined(OPENVINO_ARCH_ARM) || defined(OPENVINO_ARCH_ARM64)
     // TODO: multi-stream execution has functional issues on ARM target
+#    ifndef __APPLE__
     streamExecutorConfig._streams = 1;
```

Comment:
It should add `streamExecutorConfig._streams_changed = true;` if streams = 1 on other platforms other than MacOS.

Reply:
I have added this.